Unexpected ##[error]read ECONNRESET

https://github.com/timholy/SnoopCompile.jl/pull/107/checks?check_run_id=804747167

The error happened to me again. I am pasting the whole log so you can fix it:
https://github.com/aminya/AcuteML.jl/pull/141/checks?check_run_id=812295702
<details>
<summary>Details</summary>

2020-06-26T18:59:24.9725554Z ##[section]Starting: Request a runner to run this job
2020-06-26T18:59:25.9814713Z Can't find any online and idle self-hosted runner in current repository that matches the required labels: 'windows-latest'
2020-06-26T18:59:25.9814778Z Can't find any online and idle self-hosted runner in current repository's account/organization that matches the required labels: 'windows-latest'
2020-06-26T18:59:25.9814806Z Found online and idle hosted runner in current repository's account/organization that matches the required labels: 'windows-latest'
2020-06-26T18:59:26.1377256Z ##[section]Finishing: Request a runner to run this job
2020-06-26T18:59:33.2447746Z Current runner version: '2.263.0'
2020-06-26T18:59:33.2577681Z ##[group]Operating System
2020-06-26T18:59:33.2578152Z Microsoft Windows Server 2019
2020-06-26T18:59:33.2578303Z 10.0.17763
2020-06-26T18:59:33.2578489Z Datacenter
2020-06-26T18:59:33.2578671Z ##[endgroup]
2020-06-26T18:59:33.2578860Z ##[group]Virtual Environment
2020-06-26T18:59:33.2579103Z Environment: windows-2019
2020-06-26T18:59:33.2579291Z Version: 20200621.1
2020-06-26T18:59:33.2579563Z Included Software: https://github.com/actions/virtual-environments/blob/win19/20200621.1/images/win/Windows2019-Readme.md
2020-06-26T18:59:33.2579796Z ##[endgroup]
2020-06-26T18:59:33.2580630Z Prepare workflow directory
2020-06-26T18:59:33.3127416Z Prepare all required actions
2020-06-26T18:59:33.3147573Z Download action repository 'actions/checkout@v2'
2020-06-26T18:59:34.3130826Z Download action repository 'julia-actions/setup-julia@latest'
2020-06-26T18:59:34.4881536Z Download action repository 'julia-actions/julia-buildpkg@latest'
2020-06-26T18:59:34.7439143Z Download action repository 'julia-actions/julia-runtest@latest'
2020-06-26T18:59:35.0154115Z Download action repository 'julia-actions/julia-uploadcodecov@latest'
2020-06-26T18:59:35.9670078Z ##[group]Run actions/checkout@v2
2020-06-26T18:59:35.9670459Z with:
2020-06-26T18:59:35.9670714Z   repository: aminya/AcuteML.jl
2020-06-26T18:59:35.9671003Z   token: ***
2020-06-26T18:59:35.9671124Z   ssh-strict: true
2020-06-26T18:59:35.9671245Z   persist-credentials: true
2020-06-26T18:59:35.9671477Z   clean: true
2020-06-26T18:59:35.9671598Z   fetch-depth: 1
2020-06-26T18:59:35.9671703Z   lfs: false
2020-06-26T18:59:35.9671820Z   submodules: false
2020-06-26T18:59:35.9671941Z ##[endgroup]
2020-06-26T18:59:38.5003278Z Syncing repository: aminya/AcuteML.jl
2020-06-26T18:59:38.5292359Z ##[group]Getting Git version info
2020-06-26T18:59:38.5293223Z Working directory is 'd:\a\AcuteML.jl\AcuteML.jl'
2020-06-26T18:59:38.5692210Z [command]"C:\Program Files\Git\bin\git.exe" version
2020-06-26T18:59:38.5879235Z git version 2.27.0.windows.1
2020-06-26T18:59:38.5928639Z ##[endgroup]
2020-06-26T18:59:38.5933684Z Deleting the contents of 'd:\a\AcuteML.jl\AcuteML.jl'
2020-06-26T18:59:38.5939051Z ##[group]Initializing the repository
2020-06-26T18:59:38.5949534Z [command]"C:\Program Files\Git\bin\git.exe" init d:\a\AcuteML.jl\AcuteML.jl
2020-06-26T18:59:38.7197686Z Initialized empty Git repository in D:/a/AcuteML.jl/AcuteML.jl/.git/
2020-06-26T18:59:38.7248759Z [command]"C:\Program Files\Git\bin\git.exe" remote add origin https://github.com/aminya/AcuteML.jl
2020-06-26T18:59:38.7961545Z ##[endgroup]
2020-06-26T18:59:38.7962508Z ##[group]Disabling automatic garbage collection
2020-06-26T18:59:38.7971867Z [command]"C:\Program Files\Git\bin\git.exe" config --local gc.auto 0
2020-06-26T18:59:38.8490737Z ##[endgroup]
2020-06-26T18:59:38.8497828Z ##[group]Setting up auth
2020-06-26T18:59:38.8512247Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp core\.sshCommand
2020-06-26T18:59:38.8735919Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2020-06-26T18:59:41.2573491Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-06-26T18:59:41.2812598Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2020-06-26T18:59:41.7554141Z [command]"C:\Program Files\Git\bin\git.exe" config --local http.https://github.com/.extraheader "AUTHORIZATION: basic ***"
2020-06-26T18:59:41.7790025Z ##[endgroup]
2020-06-26T18:59:41.7833340Z ##[group]Fetching the repository
2020-06-26T18:59:41.7860003Z [command]"C:\Program Files\Git\bin\git.exe" -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +517ed0ce0199991d077ac879141adad494243725:refs/remotes/origin/SnoopCompile
2020-06-26T18:59:43.4834328Z remote: Enumerating objects: 95, done.        
2020-06-26T18:59:43.4835826Z remote: Counting objects:   1% (1/95)        
2020-06-26T18:59:43.4836270Z remote: Counting objects:   2% (2/95)        
2020-06-26T18:59:43.4840542Z remote: Counting objects:   3% (3/95)        
2020-06-26T18:59:43.4841452Z remote: Counting objects:   4% (4/95)        
2020-06-26T18:59:43.4842409Z remote: Counting objects:   5% (5/95)        
2020-06-26T18:59:43.4843252Z remote: Counting objects:   6% (6/95)        
2020-06-26T18:59:43.4843997Z remote: Counting objects:   7% (7/95)        
2020-06-26T18:59:43.4844712Z remote: Counting objects:   8% (8/95)        
2020-06-26T18:59:43.4845484Z remote: Counting objects:   9% (9/95)        
2020-06-26T18:59:43.4846174Z remote: Counting objects:  10% (10/95)        
2020-06-26T18:59:43.4846477Z remote: Counting objects:  11% (11/95)        
2020-06-26T18:59:43.4847103Z remote: Counting objects:  12% (12/95)        
2020-06-26T18:59:43.4849076Z remote: Counting objects:  13% (13/95)        
2020-06-26T18:59:43.4849425Z remote: Counting objects:  14% (14/95)        
2020-06-26T18:59:43.4850164Z remote: Counting objects:  15% (15/95)        
2020-06-26T18:59:43.4850751Z remote: Counting objects:  16% (16/95)        
2020-06-26T18:59:43.4851520Z remote: Counting objects:  17% (17/95)        
2020-06-26T18:59:43.4851789Z remote: Counting objects:  18% (18/95)        
2020-06-26T18:59:43.4851984Z remote: Counting objects:  20% (19/95)        
2020-06-26T18:59:43.4852137Z remote: Counting objects:  21% (20/95)        
2020-06-26T18:59:43.4852355Z remote: Counting objects:  22% (21/95)        
2020-06-26T18:59:43.4852519Z remote: Counting objects:  23% (22/95)        
2020-06-26T18:59:43.4852706Z remote: Counting objects:  24% (23/95)        
2020-06-26T18:59:43.4853149Z remote: Counting objects:  25% (24/95)        
2020-06-26T18:59:43.4853298Z remote: Counting objects:  26% (25/95)        
2020-06-26T18:59:43.4853431Z remote: Counting objects:  27% (26/95)        
2020-06-26T18:59:43.4853570Z remote: Counting objects:  28% (27/95)        
2020-06-26T18:59:43.4853701Z remote: Counting objects:  29% (28/95)        
2020-06-26T18:59:43.4853833Z remote: Counting objects:  30% (29/95)        
2020-06-26T18:59:43.4853964Z remote: Counting objects:  31% (30/95)        
2020-06-26T18:59:43.4854420Z remote: Counting objects:  32% (31/95)        
2020-06-26T18:59:43.4854854Z remote: Counting objects:  33% (32/95)        
2020-06-26T18:59:43.4855447Z remote: Counting objects:  34% (33/95)        
2020-06-26T18:59:43.4855841Z remote: Counting objects:  35% (34/95)        
2020-06-26T18:59:43.4856070Z remote: Counting objects:  36% (35/95)        
2020-06-26T18:59:43.4856209Z remote: Counting objects:  37% (36/95)        
2020-06-26T18:59:43.4856343Z remote: Counting objects:  38% (37/95)        
2020-06-26T18:59:43.4856481Z remote: Counting objects:  40% (38/95)        
2020-06-26T18:59:43.4856616Z remote: Counting objects:  41% (39/95)        
2020-06-26T18:59:43.4856780Z remote: Counting objects:  42% (40/95)        
2020-06-26T18:59:43.4856997Z remote: Counting objects:  43% (41/95)        
2020-06-26T18:59:43.4857164Z remote: Counting objects:  44% (42/95)        
2020-06-26T18:59:43.4857299Z remote: Counting objects:  45% (43/95)        
2020-06-26T18:59:43.4857511Z remote: Counting objects:  46% (44/95)        
2020-06-26T18:59:43.4857674Z remote: Counting objects:  47% (45/95)        
2020-06-26T18:59:43.4857807Z remote: Counting objects:  48% (46/95)        
2020-06-26T18:59:43.4858013Z remote: Counting objects:  49% (47/95)        
2020-06-26T18:59:43.4858176Z remote: Counting objects:  50% (48/95)        
2020-06-26T18:59:43.4858310Z remote: Counting objects:  51% (49/95)        
2020-06-26T18:59:43.4858515Z remote: Counting objects:  52% (50/95)        
2020-06-26T18:59:43.4858761Z remote: Counting objects:  53% (51/95)        
2020-06-26T18:59:43.4859412Z remote: Counting objects:  54% (52/95)        
2020-06-26T18:59:43.4859671Z remote: Counting objects:  55% (53/95)        
2020-06-26T18:59:43.4859831Z remote: Counting objects:  56% (54/95)        
2020-06-26T18:59:43.4859968Z remote: Counting objects:  57% (55/95)        
2020-06-26T18:59:43.4860178Z remote: Counting objects:  58% (56/95)        
2020-06-26T18:59:43.4860342Z remote: Counting objects:  60% (57/95)        
2020-06-26T18:59:43.4860474Z remote: Counting objects:  61% (58/95)        
2020-06-26T18:59:43.4860682Z remote: Counting objects:  62% (59/95)        
2020-06-26T18:59:43.4860840Z remote: Counting objects:  63% (60/95)        
2020-06-26T18:59:43.4860971Z remote: Counting objects:  64% (61/95)        
2020-06-26T18:59:43.4861177Z remote: Counting objects:  65% (62/95)        
2020-06-26T18:59:43.4861339Z remote: Counting objects:  66% (63/95)        
2020-06-26T18:59:43.4861472Z remote: Counting objects:  67% (64/95)        
2020-06-26T18:59:43.4861608Z remote: Counting objects:  68% (65/95)        
2020-06-26T18:59:43.4861818Z remote: Counting objects:  69% (66/95)        
2020-06-26T18:59:43.4862253Z remote: Counting objects:  70% (67/95)        
2020-06-26T18:59:43.4862501Z remote: Counting objects:  71% (68/95)        
2020-06-26T18:59:43.4862889Z remote: Counting objects:  72% (69/95)        
2020-06-26T18:59:43.4863204Z remote: Counting objects:  73% (70/95)        
2020-06-26T18:59:43.4863340Z remote: Counting objects:  74% (71/95)        
2020-06-26T18:59:43.4863558Z remote: Counting objects:  75% (72/95)        
2020-06-26T18:59:43.4863718Z remote: Counting objects:  76% (73/95)        
2020-06-26T18:59:43.4863850Z remote: Counting objects:  77% (74/95)        
2020-06-26T18:59:43.4864058Z remote: Counting objects:  78% (75/95)        
2020-06-26T18:59:43.4864222Z remote: Counting objects:  80% (76/95)        
2020-06-26T18:59:43.4864356Z remote: Counting objects:  81% (77/95)        
2020-06-26T18:59:43.4864570Z remote: Counting objects:  82% (78/95)        
2020-06-26T18:59:43.4864732Z remote: Counting objects:  83% (79/95)        
2020-06-26T18:59:43.4864857Z remote: Counting objects:  84% (80/95)        
2020-06-26T18:59:43.4865064Z remote: Counting objects:  85% (81/95)        
2020-06-26T18:59:43.4865226Z remote: Counting objects:  86% (82/95)        
2020-06-26T18:59:43.4865359Z remote: Counting objects:  87% (83/95)        
2020-06-26T18:59:43.4865568Z remote: Counting objects:  88% (84/95)        
2020-06-26T18:59:43.4865726Z remote: Counting objects:  89% (85/95)        
2020-06-26T18:59:43.4865856Z remote: Counting objects:  90% (86/95)        
2020-06-26T18:59:43.4866110Z remote: Counting objects:  91% (87/95)        
2020-06-26T18:59:43.4867095Z remote: Counting objects:  92% (88/95)        
2020-06-26T18:59:43.4867304Z remote: Counting objects:  93% (89/95)        
2020-06-26T18:59:43.4867540Z remote: Counting objects:  94% (90/95)        
2020-06-26T18:59:43.4867742Z remote: Counting objects:  95% (91/95)        
2020-06-26T18:59:43.4867887Z remote: Counting objects:  96% (92/95)        
2020-06-26T18:59:43.4868013Z remote: Counting objects:  97% (93/95)        
2020-06-26T18:59:43.4868200Z remote: Counting objects:  98% (94/95)        
2020-06-26T18:59:43.4868345Z remote: Counting objects: 100% (95/95)        
2020-06-26T18:59:43.4868538Z remote: Counting objects: 100% (95/95), done.        
2020-06-26T18:59:43.4872166Z remote: Compressing objects:   1% (1/86)        
2020-06-26T18:59:43.4872456Z remote: Compressing objects:   2% (2/86)        
2020-06-26T18:59:43.4873785Z remote: Compressing objects:   3% (3/86)        
2020-06-26T18:59:43.4874207Z remote: Compressing objects:   4% (4/86)        
2020-06-26T18:59:43.4874455Z remote: Compressing objects:   5% (5/86)        
2020-06-26T18:59:43.4874608Z remote: Compressing objects:   6% (6/86)        
2020-06-26T18:59:43.4874836Z remote: Compressing objects:   8% (7/86)        
2020-06-26T18:59:43.4875205Z remote: Compressing objects:   9% (8/86)        
2020-06-26T18:59:43.4875358Z remote: Compressing objects:  10% (9/86)        
2020-06-26T18:59:43.4875562Z remote: Compressing objects:  11% (10/86)        
2020-06-26T18:59:43.4875814Z remote: Compressing objects:  12% (11/86)        
2020-06-26T18:59:43.4875941Z remote: Compressing objects:  13% (12/86)        
2020-06-26T18:59:43.4876138Z remote: Compressing objects:  15% (13/86)        
2020-06-26T18:59:43.4876294Z remote: Compressing objects:  16% (14/86)        
2020-06-26T18:59:43.4876487Z remote: Compressing objects:  17% (15/86)        
2020-06-26T18:59:43.4876661Z remote: Compressing objects:  18% (16/86)        
2020-06-26T18:59:43.4876858Z remote: Compressing objects:  19% (17/86)        
2020-06-26T18:59:43.4877063Z remote: Compressing objects:  20% (18/86)        
2020-06-26T18:59:43.4877188Z remote: Compressing objects:  22% (19/86)        
2020-06-26T18:59:43.4877379Z remote: Compressing objects:  23% (20/86)        
2020-06-26T18:59:43.4877533Z remote: Compressing objects:  24% (21/86)        
2020-06-26T18:59:43.4877654Z remote: Compressing objects:  25% (22/86)        
2020-06-26T18:59:43.4877912Z remote: Compressing objects:  26% (23/86)        
2020-06-26T18:59:43.4879174Z remote: Compressing objects:  27% (24/86)        
2020-06-26T18:59:43.4879579Z remote: Compressing objects:  29% (25/86)        
2020-06-26T18:59:43.4879815Z remote: Compressing objects:  30% (26/86)        
2020-06-26T18:59:43.4880038Z remote: Compressing objects:  31% (27/86)        
2020-06-26T18:59:43.4880183Z remote: Compressing objects:  32% (28/86)        
2020-06-26T18:59:43.4880305Z remote: Compressing objects:  33% (29/86)        
2020-06-26T18:59:43.4880515Z remote: Compressing objects:  34% (30/86)        
2020-06-26T18:59:43.4880716Z remote: Compressing objects:  36% (31/86)        
2020-06-26T18:59:43.4880836Z remote: Compressing objects:  37% (32/86)        
2020-06-26T18:59:43.4881028Z remote: Compressing objects:  38% (33/86)        
2020-06-26T18:59:43.4881187Z remote: Compressing objects:  39% (34/86)        
2020-06-26T18:59:43.4881377Z remote: Compressing objects:  40% (35/86)        
2020-06-26T18:59:43.4881503Z remote: Compressing objects:  41% (36/86)        
2020-06-26T18:59:43.4881645Z remote: Compressing objects:  43% (37/86)        
2020-06-26T18:59:43.4881901Z remote: Compressing objects:  44% (38/86)        
2020-06-26T18:59:43.4882115Z remote: Compressing objects:  45% (39/86)        
2020-06-26T18:59:43.4882236Z remote: Compressing objects:  46% (40/86)        
2020-06-26T18:59:43.4882402Z remote: Compressing objects:  47% (41/86)        
2020-06-26T18:59:43.4882877Z remote: Compressing objects:  48% (42/86)        
2020-06-26T18:59:43.4883094Z remote: Compressing objects:  50% (43/86)        
2020-06-26T18:59:43.4884034Z remote: Compressing objects:  51% (44/86)        
2020-06-26T18:59:43.4884597Z remote: Compressing objects:  52% (45/86)        
2020-06-26T18:59:43.4885206Z remote: Compressing objects:  53% (46/86)        
2020-06-26T18:59:43.4885839Z remote: Compressing objects:  54% (47/86)        
2020-06-26T18:59:43.4886099Z remote: Compressing objects:  55% (48/86)        
2020-06-26T18:59:43.4886321Z remote: Compressing objects:  56% (49/86)        
2020-06-26T18:59:43.4890975Z remote: Compressing objects:  58% (50/86)        
2020-06-26T18:59:43.4891265Z remote: Compressing objects:  59% (51/86)        
2020-06-26T18:59:43.4891543Z remote: Compressing objects:  60% (52/86)        
2020-06-26T18:59:43.4897430Z remote: Compressing objects:  61% (53/86)        
2020-06-26T18:59:43.4897600Z remote: Compressing objects:  62% (54/86)        
2020-06-26T18:59:43.4902558Z remote: Compressing objects:  63% (55/86)        
2020-06-26T18:59:43.4902732Z remote: Compressing objects:  65% (56/86)        
2020-06-26T18:59:43.4902949Z remote: Compressing objects:  66% (57/86)        
2020-06-26T18:59:43.4904683Z remote: Compressing objects:  67% (58/86)        
2020-06-26T18:59:43.4910800Z remote: Compressing objects:  68% (59/86)        
2020-06-26T18:59:43.4913539Z remote: Compressing objects:  69% (60/86)        
2020-06-26T18:59:43.4913713Z remote: Compressing objects:  70% (61/86)        
2020-06-26T18:59:43.4916609Z remote: Compressing objects:  72% (62/86)        
2020-06-26T18:59:43.4916834Z remote: Compressing objects:  73% (63/86)        
2020-06-26T18:59:43.4916987Z remote: Compressing objects:  74% (64/86)        
2020-06-26T18:59:43.4924935Z remote: Compressing objects:  75% (65/86)        
2020-06-26T18:59:43.4925169Z remote: Compressing objects:  76% (66/86)        
2020-06-26T18:59:43.4925369Z remote: Compressing objects:  77% (67/86)        
2020-06-26T18:59:43.4925464Z remote: Compressing objects:  79% (68/86)        
2020-06-26T18:59:43.4925650Z remote: Compressing objects:  80% (69/86)        
2020-06-26T18:59:43.4925834Z remote: Compressing objects:  81% (70/86)        
2020-06-26T18:59:43.4925942Z remote: Compressing objects:  82% (71/86)        
2020-06-26T18:59:43.4926125Z remote: Compressing objects:  83% (72/86)        
2020-06-26T18:59:43.4926284Z remote: Compressing objects:  84% (73/86)        
2020-06-26T18:59:43.4926552Z remote: Compressing objects:  86% (74/86)        
2020-06-26T18:59:43.4926666Z remote: Compressing objects:  87% (75/86)        
2020-06-26T18:59:43.4926843Z remote: Compressing objects:  88% (76/86)        
2020-06-26T18:59:43.4927018Z remote: Compressing objects:  89% (77/86)        
2020-06-26T18:59:43.4927125Z remote: Compressing objects:  90% (78/86)        
2020-06-26T18:59:43.4927296Z remote: Compressing objects:  91% (79/86)        
2020-06-26T18:59:43.4927480Z remote: Compressing objects:  93% (80/86)        
2020-06-26T18:59:43.4927586Z remote: Compressing objects:  94% (81/86)        
2020-06-26T18:59:43.4927756Z remote: Compressing objects:  95% (82/86)        
2020-06-26T18:59:43.4927942Z remote: Compressing objects:  96% (83/86)        
2020-06-26T18:59:43.4928049Z remote: Compressing objects:  97% (84/86)        
2020-06-26T18:59:43.4928221Z remote: Compressing objects:  98% (85/86)        
2020-06-26T18:59:43.4928350Z remote: Compressing objects: 100% (86/86)        
2020-06-26T18:59:43.4928528Z remote: Compressing objects: 100% (86/86), done.        
2020-06-26T18:59:43.6161760Z remote: Total 95 (delta 6), reused 30 (delta 1), pack-reused 0        
2020-06-26T18:59:43.7299025Z From https://github.com/aminya/AcuteML.jl
2020-06-26T18:59:43.7311825Z  * [new ref]         517ed0ce0199991d077ac879141adad494243725 -> origin/SnoopCompile
2020-06-26T18:59:43.7752003Z ##[endgroup]
2020-06-26T18:59:43.7753718Z ##[group]Determining the checkout info
2020-06-26T18:59:43.7754377Z ##[endgroup]
2020-06-26T18:59:43.7754656Z ##[group]Checking out the ref
2020-06-26T18:59:43.7765648Z [command]"C:\Program Files\Git\bin\git.exe" checkout --progress --force -B SnoopCompile refs/remotes/origin/SnoopCompile
2020-06-26T18:59:43.8573922Z Branch 'SnoopCompile' set up to track remote branch 'SnoopCompile' from 'origin'.
2020-06-26T18:59:43.8574493Z Switched to a new branch 'SnoopCompile'
2020-06-26T18:59:43.8615751Z ##[endgroup]
2020-06-26T18:59:43.8625931Z [command]"C:\Program Files\Git\bin\git.exe" log -1
2020-06-26T18:59:43.8823764Z commit 517ed0ce0199991d077ac879141adad494243725
2020-06-26T18:59:43.8824403Z Author: Amin Yahyaabadi <aminyahyaabadi74@gmail.com>
2020-06-26T18:59:43.8824764Z Date:   Fri Jun 26 13:59:16 2020 -0500
2020-06-26T18:59:43.8824926Z 
2020-06-26T18:59:43.8825118Z     yml typo
2020-06-26T18:59:43.9091439Z ##[group]Run julia-actions/setup-julia@latest
2020-06-26T18:59:43.9091607Z with:
2020-06-26T18:59:43.9091718Z   version: 1.2
2020-06-26T18:59:43.9091821Z   arch: x64
2020-06-26T18:59:43.9091924Z   show-versioninfo: false
2020-06-26T18:59:43.9092030Z ##[endgroup]
2020-06-26T18:59:44.2635433Z ##[error]read ECONNRESET
2020-06-26T18:59:44.2854891Z Post job cleanup.
2020-06-26T18:59:46.0545848Z [command]"C:\Program Files\Git\bin\git.exe" version
2020-06-26T18:59:46.0724734Z git version 2.27.0.windows.1
2020-06-26T18:59:46.0824461Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp core\.sshCommand
2020-06-26T18:59:46.1047061Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2020-06-26T18:59:46.5676836Z [command]"C:\Program Files\Git\bin\git.exe" config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-06-26T18:59:46.5851138Z http.https://github.com/.extraheader
2020-06-26T18:59:46.5895240Z [command]"C:\Program Files\Git\bin\git.exe" config --local --unset-all http.https://github.com/.extraheader
2020-06-26T18:59:46.6127678Z [command]"C:\Program Files\Git\bin\git.exe" submodule foreach --recursive "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2020-06-26T18:59:47.1161047Z Cleaning up orphan processes

</details>

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -22,6 +22,7 @@ const crypto = __importStar(require("crypto"));
 const fs = __importStar(require("fs"));
 const os = __importStar(require("os"));
 const path = __importStar(require("path"));
+const async_retry_1 = require("async-retry");
 const semver = __importStar(require("semver"));
 // Translations between actions input and Julia arch names
 const osMap = {
@@ -59,7 +60,15 @@ function calculateChecksum(file) {
  */
 function getJuliaVersionInfo() {
     return __awaiter(this, void 0, void 0, function* () {
-        const versionsFile = yield tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json');
+        // Occasionally the connection is reset for unknown reasons
+        // In those cases, retry the download
+        const versionsFile = yield async_retry_1.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+            return yield tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json');
+        }), {
+            onRetry: (err) => {
+                core.debug(`Download of versions.json failed, trying again. Error: ${err}`);
+            }
+        });
         return JSON.parse(fs.readFileSync(versionsFile).toString());
     });
 }
@@ -146,7 +155,15 @@ function installJulia(versionInfo, version, arch) {
         const fileInfo = getFileInfo(versionInfo, version, arch);
         const downloadURL = getDownloadURL(fileInfo, version, arch);
         core.debug(`downloading Julia from ${downloadURL}`);
-        const juliaDownloadPath = yield tc.downloadTool(downloadURL);
+        // Occasionally the connection is reset for unknown reasons
+        // In those cases, retry the download
+        const juliaDownloadPath = yield async_retry_1.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+            return yield tc.downloadTool(downloadURL);
+        }), {
+            onRetry: (err) => {
+                core.debug(`Download of ${downloadURL} failed, trying again. Error: ${err}`);
+            }
+        });
         // Verify checksum
         if (!version.endsWith('nightly')) {
             const checkSum = yield calculateChecksum(juliaDownloadPath);

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -22,7 +22,7 @@ const crypto = __importStar(require("crypto"));
 const fs = __importStar(require("fs"));
 const os = __importStar(require("os"));
 const path = __importStar(require("path"));
-const async_retry_1 = require("async-retry");
+const retry = __importStar(require("async-retry"));
 const semver = __importStar(require("semver"));
 // Translations between actions input and Julia arch names
 const osMap = {
@@ -62,7 +62,7 @@ function getJuliaVersionInfo() {
     return __awaiter(this, void 0, void 0, function* () {
         // Occasionally the connection is reset for unknown reasons
         // In those cases, retry the download
-        const versionsFile = yield async_retry_1.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+        const versionsFile = yield retry.retry((bail) => __awaiter(this, void 0, void 0, function* () {
             return yield tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json');
         }), {
             onRetry: (err) => {
@@ -157,7 +157,7 @@ function installJulia(versionInfo, version, arch) {
         core.debug(`downloading Julia from ${downloadURL}`);
         // Occasionally the connection is reset for unknown reasons
         // In those cases, retry the download
-        const juliaDownloadPath = yield async_retry_1.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+        const juliaDownloadPath = yield retry.retry((bail) => __awaiter(this, void 0, void 0, function* () {
             return yield tc.downloadTool(downloadURL);
         }), {
             onRetry: (err) => {

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -22,7 +22,7 @@ const crypto = __importStar(require("crypto"));
 const fs = __importStar(require("fs"));
 const os = __importStar(require("os"));
 const path = __importStar(require("path"));
-const retry = __importStar(require("async-retry"));
+const retry = require("async-retry");
 const semver = __importStar(require("semver"));
 // Translations between actions input and Julia arch names
 const osMap = {
@@ -62,7 +62,7 @@ function getJuliaVersionInfo() {
     return __awaiter(this, void 0, void 0, function* () {
         // Occasionally the connection is reset for unknown reasons
         // In those cases, retry the download
-        const versionsFile = yield retry.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+        const versionsFile = yield retry((bail) => __awaiter(this, void 0, void 0, function* () {
             return yield tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json');
         }), {
             onRetry: (err) => {
@@ -157,7 +157,7 @@ function installJulia(versionInfo, version, arch) {
         core.debug(`downloading Julia from ${downloadURL}`);
         // Occasionally the connection is reset for unknown reasons
         // In those cases, retry the download
-        const juliaDownloadPath = yield retry.retry((bail) => __awaiter(this, void 0, void 0, function* () {
+        const juliaDownloadPath = yield retry((bail) => __awaiter(this, void 0, void 0, function* () {
             return yield tc.downloadTool(downloadURL);
         }), {
             onRetry: (err) => {

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
     "semver": "^6.3.0"
   },
   "devDependencies": {
+    "@types/async-retry": "^1.4.2",
     "@types/jest": "^24.0.13",
     "@types/node": "^12.12.7",
     "@types/semver": "^6.0.0",

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
     "@actions/exec": "^1.0.0",
     "@actions/io": "^1.0.0",
     "@actions/tool-cache": "^1.0.0",
+    "async-retry": "^1.3.1",
     "semver": "^6.3.0"
   },
   "devDependencies": {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -6,6 +6,7 @@ import * as crypto from 'crypto'
 import * as fs from 'fs'
 import * as os from 'os'
 import * as path from 'path'
+import { retry } from 'async-retry'
 
 import * as semver from 'semver'
 
@@ -47,7 +48,15 @@ async function calculateChecksum(file: string): Promise<string> {
  * @returns The content of the downloaded versions.json file as object.
  */
 export async function getJuliaVersionInfo(): Promise<object> {
-    const versionsFile = await tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json')
+    // Occasionally the connection is reset for unknown reasons
+    // In those cases, retry the download
+    const versionsFile = await retry(async (bail: Function) => {
+        return await tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json')
+    }, {
+        onRetry: (err: Error) => {
+            core.debug(`Download of versions.json failed, trying again. Error: ${err}`)
+        }
+    })
 
     return JSON.parse(fs.readFileSync(versionsFile).toString())
 }
@@ -141,7 +150,17 @@ export async function installJulia(versionInfo, version: string, arch: string): 
     const fileInfo = getFileInfo(versionInfo, version, arch)
     const downloadURL = getDownloadURL(fileInfo, version, arch)
     core.debug(`downloading Julia from ${downloadURL}`)
-    const juliaDownloadPath = await tc.downloadTool(downloadURL)
+
+    // Occasionally the connection is reset for unknown reasons
+    // In those cases, retry the download
+    const juliaDownloadPath = await retry(async (bail: Function) => {
+        return await tc.downloadTool(downloadURL)
+    }, {
+        onRetry: (err: Error) => {
+            core.debug(`Download of ${downloadURL} failed, trying again. Error: ${err}`)
+        }
+    })
+
 
     // Verify checksum
     if (!version.endsWith('nightly')) {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -6,7 +6,7 @@ import * as crypto from 'crypto'
 import * as fs from 'fs'
 import * as os from 'os'
 import * as path from 'path'
-import * as retry from 'async-retry'
+import retry = require('async-retry')
 
 import * as semver from 'semver'
 
@@ -50,7 +50,7 @@ async function calculateChecksum(file: string): Promise<string> {
 export async function getJuliaVersionInfo(): Promise<object> {
     // Occasionally the connection is reset for unknown reasons
     // In those cases, retry the download
-    const versionsFile = await retry.retry(async (bail: Function) => {
+    const versionsFile = await retry(async (bail: Function) => {
         return await tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json')
     }, {
         onRetry: (err: Error) => {
@@ -153,7 +153,7 @@ export async function installJulia(versionInfo, version: string, arch: string): 
 
     // Occasionally the connection is reset for unknown reasons
     // In those cases, retry the download
-    const juliaDownloadPath = await retry.retry(async (bail: Function) => {
+    const juliaDownloadPath = await retry(async (bail: Function) => {
         return await tc.downloadTool(downloadURL)
     }, {
         onRetry: (err: Error) => {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -6,7 +6,7 @@ import * as crypto from 'crypto'
 import * as fs from 'fs'
 import * as os from 'os'
 import * as path from 'path'
-import { retry } from 'async-retry'
+import * as retry from 'async-retry'
 
 import * as semver from 'semver'
 
@@ -50,7 +50,7 @@ async function calculateChecksum(file: string): Promise<string> {
 export async function getJuliaVersionInfo(): Promise<object> {
     // Occasionally the connection is reset for unknown reasons
     // In those cases, retry the download
-    const versionsFile = await retry(async (bail: Function) => {
+    const versionsFile = await retry.retry(async (bail: Function) => {
         return await tc.downloadTool('https://julialang-s3.julialang.org/bin/versions.json')
     }, {
         onRetry: (err: Error) => {
@@ -153,7 +153,7 @@ export async function installJulia(versionInfo, version: string, arch: string): 
 
     // Occasionally the connection is reset for unknown reasons
     // In those cases, retry the download
-    const juliaDownloadPath = await retry(async (bail: Function) => {
+    const juliaDownloadPath = await retry.retry(async (bail: Function) => {
         return await tc.downloadTool(downloadURL)
     }, {
         onRetry: (err: Error) => {

